nautobot/extras/migrations/0054_scheduledjob_kwargs_request_user_change.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+
+def rename_scheduled_job_kwargs_request_user(apps, schema_editor):
+    ScheduledJob = apps.get_model("extras", "ScheduledJob")
+    for sj in ScheduledJob.objects.all():
+        if "request" in sj.kwargs and sj.kwargs["request"].get("user", None) is not None:
```

Comment:
Would `ScheduledJob.kwargs["request"]["user"]` ever be set to None? Should we just be looking for `if "request" in sj.kwargs and "user" in sj.kwargs["request"]:`

Reply:
Changes in 511498e resolved this and was validated.